src/main/java/io/javalin/embeddedserver/jetty/websocket/WebSocketCreators.kt

```diff
@@ -18,8 +18,12 @@ class JettyWebSocketCreator(internal val handler: Any) : WebSocketCreator {
 
 class RootWebSocketCreator(private val handlerRoot: WebSocketHandlerRoot, private val javalinWsHandlers: List<WebSocketHandler>) : WebSocketCreator {
     override fun createWebSocket(req: ServletUpgradeRequest, response: ServletUpgradeResponse): Any {
-        if (javalinWsHandlers.find { it.matches(req.requestURI.path) } == null) {
+        val handler: WebSocketHandler? = javalinWsHandlers.find { it.matches(req.requestURI.path) }
+
+        if (handler == null) {
```

Comment:
```kotlin
handler?.updateParams(req.requestURI.path) ?: response.sendError()
```

Reply:
Thanks for your review, but with the last changes that @tipsy requested, it's easier to do this in `onConnect`